andela/mrm_api
# #166209894 temporarily remove old analytics queries

#### What does this PR do?
Remove old analytics queries to allow testing of the new consolidated analytics query for optimization

#### Description of Task to be completed?
- Remove old analytics queries
- Remove old analytics test cases.

#### How should this be manually tested?
git pull and checkout to the branch `ch-disable-old-analytics-queries-166209894` 
run application
Run any of the old analytics queries, it should return null.

#### What are the relevant pivotal tracker stories?
[#166209894](https://www.pivotaltracker.com/story/show/166209894)
